Poker.BE/Poker.BE.Domain.Tests/Game/HandTests.cs

```diff
@@ -11,10 +11,85 @@ namespace Poker.BE.Domain.Game.Tests
     [TestClass]
     public class HandTests
     {
-        [TestMethod]
-        public void DealCardsTest()
+        [TestMethod()]
+        public void PrepareHandTest()
```

Comment:
❓ i didn't understand why `Hand.PrepereHand()` is `public` instead of `private`.